test_conformance/extensions/cl_khr_kernel_clock/procs.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2024 The Khronos Group Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef _CL_KHR_KERNEL_CLOCK_PROCS_H
+#define _CL_KHR_KERNEL_CLOCK_PROCS_H
```

Comment:
```suggestion
#ifndef CL_KHR_KERNEL_CLOCK_PROCS_H
#define CL_KHR_KERNEL_CLOCK_PROCS_H
```

Each name that begins with an underscore followed by an uppercase letter is reserved.

Reply:
Done